tests/install-drupal.sh

```
@@ -0,0 +1,87 @@
+    exit 0
+fi
+
+# Set proper permissions
+echo "Setting up permissions..."
+docker compose exec -T $SERVICE sh -c "mkdir -p ${WEBROOT}/web/sites/default/files && chmod -R 777 ${WEBROOT}/web/sites/default/files"
+docker compose exec -T $SERVICE sh -c "chmod 777 ${WEBROOT}/web/sites/default"
+
+# Install Drupal using drush with SQLite database file
+echo "Installing Drupal using drush with SQLite..."
+docker compose exec -T $SERVICE sh -c "cd ${WEBROOT} && vendor/bin/drush site:install minimal \
+    --db-url="sqlite://localhost/sites/default/files/.ht.sqlite" \
```

Shellcheck (0.11.0)

[warning] Word is of the form "A"B"C" (B indicated). Did you mean "ABC" or "A\"B\"C"?

(SC2140)